supervisor/const.py

```diff
@@ -423,6 +412,21 @@ class AddonState(StrEnum):
     ERROR = "error"
 
 
+class MappingType(StrEnum):
```

Comment:
Copy this class into `addons/const.py` - we migrate away from this big single const.py file